YourToDo/templates/planner/planner.html

```diff
@@ -0,0 +1,208 @@
+                        <span id="planner-date-month-selector"></span>
+                        <span class="glyphicon glyphicon-chevron-right planner-date-selectors-button" onclick="changeViewDate('MONTH', 1)"></span>
+                    </div>
+
+                    <!-- Weekly dates (i.e. "< 11/15 - 11/22 >") -->
+                    <div class="planner-date-week-selector">
+                        <span class="glyphicon glyphicon-chevron-left planner-date-selectors-button" onclick="changeViewDate('WEEK', -1)"></span>
+                        <span id="planner-date-week-selector"></span>
+                        <span class="glyphicon glyphicon-chevron-right planner-date-selectors-button" onclick="changeViewDate('WEEK', 1)"></span>
+                    </div>
+
+                    <!-- Years (i.e. "< 2015 >" ) -->
+                    <div class="planner-date-year-selector">
+                        <span class="glyphicon glyphicon-chevron-left planner-date-selectors-button" onclick="changeViewDate('YEAR', -1)"></span>
+                        <span id="planner-date-year-selector"></span>
+                        <span class="glyphicon glyphicon-chevron-right planner-date-selectors-button" onclick="changeViewDate('YEAR', 1)"></span>
+                    </div>
+                </td>
+                <td class="planner-layout-selectors" colspan="2">
+                    <div class="btn-group" role="group">
+                        <button type="button" class="btn btn-default btn-layout-selector" id="btn-layout-day">Day</button>
+                        <button type="button" class="btn btn-default btn-layout-selector active" id="btn-layout-week">Week</button>
+                        <button type="button" class="btn btn-default btn-layout-selector" id="btn-layout-month">Month</button>
+                    </div>
+                </td>
+            </tr>
+            
+            <!-- Days List -->
+            <tr class="planner-week-days">
+                <td>Sunday</td>
+                <td>Monday</td>
+                <td>Tuesday</td>
+                <td>Wednesday</td>
+                <td>Thurday</td>
+                <td>Friday</td>
+                <td>Saturday</td>
+            </tr><!-- End Top Content -->
+        </table>
+
+        <!-- TODO: Add custom scrollbars that don't look awful -->
+
+        <div class="planner-week-content">
+            <table class="planner-container">
+                <!-- Column Specification: Left sidebar, 7 days -->
+                <col class="planner-week-left-sidebar">
+                <col class="planner-week-days-day">
+                <col class="planner-week-days-day">
+                <col class="planner-week-days-day">
+                <col class="planner-week-days-day">
+                <col class="planner-week-days-day">
+                <col class="planner-week-days-day">
+                <col class="planner-week-days-day">
+
+                <tr class="planner-week-row">
```

Comment:
Obviously these will be removed when we actually call the categories from the database.